addons/binding/org.openhab.binding.tankerkoenig/src/main/java/org/openhab/binding/tankerkoenig/internal/config/OpeningTime.java

```diff
@@ -47,4 +47,14 @@ public void setEnd(String end) {
         this.end = end;
     }
 
+    @Override
+    public String toString() {
+        StringBuilder sb = new StringBuilder();
+        sb.append(this.getText());
+        sb.append(" Open: ");
```

Comment:
Just a side note: the StringBuilder.append() method returns itself and could be used for method chaining.

`return new StringBuilder().append(...).append(...)....to String();`

Reply:
Learned something. Thanks.